Flipkart-price-alert/README.md

```diff
@@ -0,0 +1,22 @@
+# Flipkart Price Alert
+Checks for the price of a desired product periodically.
+If the price drops below the desired amount, notifies the user via email.
+# Setup Instructions
+Type the following to install requirements: (make sure you have python installed)
+`pip install -r requirements.txt`
+This script requires access to mail credentials inorder to work, follow the steps below to generate the credentials for a limited scope.
+## How to get app password
```

Comment:
It's confusing to write "app passwords". I would suggest rewriting it as "Optional: How to get alternate login credentials"